app/test-utils/testFunctions/questionTests/errorTests.js

```diff
@@ -86,7 +86,8 @@ const errorTests = ({
       .expect(errorSummaryList.find('li:nth-child(1)').innerText).eql(expectedErrorMessage)
       .expect(errorSummaryList.find('li:nth-child(1) a').getAttribute('href')).eql(`${expectedAnchorLink}`)
       .expect(renderedQuestion.exists).ok()
-      .expect(renderedQuestion.find('.nhsuk-error-message').innerText).eql(`Error:\n${expectedErrorMessage}`);
+      .expect(renderedQuestion.find('.nhsuk-error-message').innerText).contains('Error:')
+      .expect(renderedQuestion.find('.nhsuk-error-message').innerText).contains(expectedErrorMessage);
```

Comment:
The pipeline tests are showing as a space and locally as \n so I have split this test into 2 parts. Also had to use contains instead pf trimming as it is found after the button is clicked so cannot be found at the start of the tests.

Reply:
👍